src/py/flwr/server/app.py

```diff
@@ -246,6 +262,11 @@ def run_superlink() -> None:
     # Obtain certificates
     certificates = try_obtain_server_certificates(args, args.fleet_api_type)
 
+    maybe_config = _try_obtain_config(args)
+    auth_plugin: Optional[ExecAuthPlugin] = None
+    if maybe_config is not None:
+        auth_plugin = _try_obtain_exec_auth_plugin(maybe_config)
```

Comment:
We must raise an error here. When running `flower-superlink` with user auth config but the auth plugin can't be loaded, this is surely an error case and we should not let the SuperLink start without user auth.

Reply:
False alarm.